osbrain/tests/test_agent.py

```diff
@@ -105,6 +105,88 @@ def test_agent_shutdown(nsproxy):
     assert 'a0' not in nsproxy.list()
 
```

Comment:
I would maybe rename all tests as `test_close_ipc_socket_xxxx` (adding the `ipc`).

Maybe we could even create a separate file for IPC tests (that would simplify the test suite Windows compatibility too).

Reply:
Yes, I was thinking about putting them in a separate file, but it felt too drastic in the end, so I kept them in this one 😄 
I'll move them in the next fixup.